tests/test_util_nodes.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
Review is **NOT** needed.
Converting yield testing into pytest.mark.parametrize